glide.yaml

```diff
@@ -0,0 +1,4 @@
+package: github.com/theplant/hsm2-backend/vendor/github.com/theplant/resources
```

Comment:
Why `package` name is under `hsm2-backend/vendor`? Shouldn't it be `github.com/theplant/resources`.